apitools/base/protorpclite/test_util.py

```diff
@@ -579,11 +579,17 @@ def testContentType(self):
         self.assertTrue(isinstance(self.PROTOLIB.CONTENT_TYPE, str))
 
     def testDecodeInvalidEnumType(self):
-        self.assertRaisesWithRegexpMatch(messages.DecodeError,
-                                         'Invalid enum value ',
-                                         self.PROTOLIB.decode_message,
-                                         OptionalMessage,
-                                         self.encoded_invalid_enum)
+        # Since protos need to be able to add new enums, a message should be
+        # successfully decoded even if the enum value is invalid. Encoding the
+        # decoded message should result in equivalence with the original
+        # encoded message containing an invalid enum.
+        decoded = self.PROTOLIB.decode_message(
+            OptionalMessage, self.encoded_invalid_enum)
+        message = OptionalMessage()
+        self.assertEqual(message, decoded)
+        encoded = self.PROTOLIB.encode_message(
```

Comment:
Nit: This line break seems silly

Reply:
No, sorry, just copy-pasting without reformatting I think. Went through and condensed a bit, please take a look.